### DISCUSSION TITLE
Add "Feature Request" board

### DISCUSSION THREAD

**Discussion by corysimmons** [hcengineering/platform#4595] [Ideas]
Something like Canny where users can upvote feature requests and it can be added to sprints seamlessly.

That's it. All your features + replace Intercom + replace Canny = I will replace every tool with this.

**Comment by aplatoff** [comment]
Absolutely, let's complete the cycle! Thank you!

**Reply by corysimmons** [reply]
Similar to the other thread, just having a swimlane like "From Feature Requests" that a PM can triage would be cool.

**Reply by aplatoff** [reply]
Let's do it!
